src/components/pill/mastercard.js

```diff
@@ -5,7 +5,18 @@ import { MasterCardContainer } from "./styled"
 
 const MasterCard = forwardRef(
   (
-    { children, height, normal, onClick, pillLeft = {}, pillRight = {}, round, size, ...rest },
+    {
+      children,
+      height,
+      normal,
+      onClick,
+      pillLeft = {},
+      pillRight = {},
+      round,
+      size,
+      "data-testid": dataTestId,
```

Comment:
Maybe we could have `testId` here to avoid this! :)

Reply:
i thought so at beginning, but on the other hand, for all the other components we add prop with dashes, ie.`<Link data-testid="dropdown-node-link"` so if we change the pattern now, you will need to remember which uses which. And this "data-testid": is written only once, so i thought it will be easier just to follow naming we know
WDYT?

Reply:
I'm fine with it, just mentioned it for more convenience! 😄